addons/project_todo/__manifest__.py

```diff
@@ -0,0 +1,38 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+{
+    'name': 'To-Do',
+    'version': '1.0',
+    'category': 'Productivity/To-Do',
+    # TODO: A web page has to be created for To-do
```

Comment:
I think you can remove the todo, we will just inform Alexandra to inform the marketing team (I think you have already informed her). 🤔 🙂